i-RIC/iriclib
# Close #86

To test this, please do the followings:

1. Install Miniconda. It is available from the following URL. For example, install to D:\miniconda3.
https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe
2. Install swig, and add the folder where swig.exe is installed, to PATH environmental variable. It is available from the following URL:
http://www.swig.org/
3. Build iriclib.dll, as usual.
4. "Open Anaconda Prompt (miniconda3)", and enter the following command, to create virtual env named "iric".
```
conda create -n "iric" python=3
```
3. Modify python_binding/build.bat, so that it can include Python.h, cgnslib.h, and build and link against python.
4. Copy the following files to D:\miniconda3\envs\iric\Lib\site-packages (or where you've installed miniconda):

    * iriclib.dll
    * cgnslib.dll
    * hdf5.dll
    * szip.dll
    * zlib.dll

5. In "Anaconda Prompt (miniconda3)", input the following commands, to make sure that iric module is installed properly.
```
conda activate iric
python
import iric
```

6. Please run unit tests with the following command:
```
cd (Where iriclib exists)\python_binding\unittests_cgnsfile
python run.py
